src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -184,6 +206,7 @@ public TestCaseView createTestCaseView(
         .createStubFunctionName(namer.getCreateStubFunctionName(methodContext.getTargetInterface()))
         .grpcStubCallString(namer.getGrpcStubCallString(methodContext.getTargetInterface(), method))
         .clientHasDefaultInstance(methodContext.getInterfaceConfig().hasDefaultInstance())
+        .methodDescriptor(methodDescriptorName)
```

Comment:
I'm uneasy about this particular design of using the method descriptor from the stub. I'm going to think about it more as I look at your other PRs.

Reply:
Ok.

For https://github.com/googleapis/gax-java/pull/493, this would changed to use the response serializer from the stub instead of the the entire methodDescriptor. The current unit test design includes faking everything server-side, which includes serializing the response object into an HTTP response, so the response serializer needs to be given to the MockHttpService.